sys/Makefile.include

```diff
@@ -73,6 +73,10 @@ ifneq (,$(filter oneway_malloc,$(USEMODULE)))
     USEMODULE_INCLUDES += $(RIOTBASE)/sys/oneway-malloc/include
 endif
 
+ifneq (,$(filter fib,$(USEMODULE)))
+    USEMODULE_INCLUDES += $(RIOTBASE)/sys/net/include
```

Comment:
move `fib/fib.h` to `sys/include/net/ng_fib.h` and all its sub-headers to `sys/include/net/ng_fib/` and remove this line to make this part of NTSF.